avocado/core/settings.py

```diff
@@ -40,7 +40,8 @@
 _config_dir_system_extra = os.path.join(CFG_DIR, 'avocado', 'conf.d')
 _config_dir_local = os.path.join(USER_DIR, '.config', 'avocado')
 _source_tree_root = os.path.join(sys.modules[__name__].__file__, "..", "..", "..")
-_config_path_intree = os.path.join(os.path.abspath(_source_tree_root), 'etc', 'avocado')
+_config_path_intree = os.path.join(os.path.abspath(_source_tree_root),
+                                   'avocado', 'etc', 'avocado')
```

Comment:
Wouldn't more appropriate location be `/usr/share/avocado/conf/`? My suggestion comes from Xorg (`/usr/share/X11/xorg.conf.d/10-quirks.conf`).

Reply:
I think the Xorg example is an exception to the rule, and still, it also allows for configuration files in `/etc`.  Also, the change above is really about the "in tree" location, which moved to be a part of the module data.